fv3config/_datastore.py

```diff
@@ -2,37 +2,45 @@
 import tarfile
 import shutil
 import logging
-import appdirs
+import tempfile
 from subprocess import check_call
+import requests
+import appdirs
 from ._exceptions import ConfigError, DataMissingError
-try:
-    import wget
-except ImportError:
-    wget = None
-    import requests
-
-app_name = 'fv3gfs'
-app_author = 'vulcan'
-
-app_data_dir = appdirs.user_data_dir(app_name, app_author)
-local_archive_dir = os.path.join(app_data_dir, 'archive')
-
-filename = '2019-10-23-data-for-running-fv3gfs.tar.gz'
-filename_root = '2019-10-23-data-for-running-fv3gfs'
-url = f'http://storage.googleapis.com/vcm-ml-public/{filename}'
-local_archive_filename = os.path.join(app_data_dir, filename)
-gs_bucket_prefix = 'gs://'
-
-forcing_options_dict = {
-    'default': os.path.join(local_archive_dir, 'base_forcing')
+
+if 'FV3CONFIG_CACHE_DIR' in os.environ:
+    LOCAL_ARCHIVE_DIR = os.environ['FV3CONFIG_CACHE_DIR']
+else:
+    LOCAL_ARCHIVE_DIR = os.path.join(
```

Comment:
If using the default, you impose some sub-structure to this directory (using appdirs). Would it make sense to impose the same substructure to the user-specified directory?

Reply:
I don't think so, the appdirs arguments are just used to find the system-located directory for specific uses (like `/username/.cache/vulcan/fv3gfs/`). Within a directory the user has specified to use for fv3config cache, it shouldn't be necessary to specify that this is fv3config or made by vulcan. We only have any kind of internal subdirectory for safety in edge cases (e.g. when the user uses `$SCRATCH` or something like that).